src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/RabbitMqBasicConsumer.cs

```diff
@@ -128,13 +121,7 @@ public void HandleBasicCancelOk(string consumerTag)
 
             LogContext.Debug?.Log("Consumer Cancel Ok: {InputAddress} - {ConsumerTag}", _context.InputAddress, consumerTag);
 
-            if (_dispatcher.ActiveDispatchCount == 0)
-            {
-                _deliveryComplete.TrySetResult(true);
-                SetCompleted(Task.CompletedTask);
-            }
-            else
-                SetCompleted(_deliveryComplete.Task);
+            TrySetConsumeCanceled();
```

Comment:
This shouldn't be "canceled", if we called BasicCancel, this is simply the acknowledgement of that call by the broker. So this is a clean-shutdown of the consumer ack'd by the broker. Not canceled.